src/navigation/checkout/Search.js

```diff
@@ -180,9 +179,7 @@ class RestaurantsPage extends Component {
       );
     } else {
       return (
-        <SafeAreaView
-          edges={['right', 'bottom', 'left']}
-          style={{ flexGrow: 1 }}>
+        <View edges={['right', 'bottom', 'left']} style={{ flexGrow: 1 }}>
```

Comment:
I believe there is no `edges` property on the `View`. If `SafeAreaView` is not needed here, I guess it can be also removed